templates/vhost/locations/proxy.erb

```diff
@@ -1,3 +1,4 @@
+<% if @proxy -%>
     proxy_pass            <%= @proxy %>;
     proxy_read_timeout    <%= @proxy_read_timeout %>;
     proxy_connect_timeout <%= @proxy_connect_timeout %>;
```

Comment:
@ubellavance Only oversight.  Feel free to send a PR for any change that needs to be made here.